src/main/kotlin/com/amazon/ion/plugin/intellij/Ion.bnf

```diff
@@ -21,11 +21,13 @@ struct ::= LBRACE [members] RBRACE {pin=1}
 members ::= struct_pair*
 pair ::= COMMENT* (key SEPARATOR value)
 key ::= (QUOTE KEY_NAME QUOTE) | (QQUOTE KEY_NAME QQUOTE) | (QQQUOTE KEY_NAME QQQUOTE) | KEY_NAME {methods=[getKeyName]}
-list ::= LBRACKET [elements] RBRACKET {pin=1}
-elements ::= list_element*
-sexpression ::= (LPAREN [sexpression_operator] [atoms] RPAREN)
+list ::= LBRACKET [list_elements] RBRACKET {pin=1}
+list_elements ::= list_element*
+sexpression ::= (LPAREN [sexpression_elements] RPAREN)
+sexpression_elements ::= [sexpression_operator] atoms*
 sexpression_operator ::= (annotation)* (symbol)
-atoms ::= sexpression_atom+
+sexpression_atom ::= OPERATOR | value {pin(".*")=1 recoverWhile=not_paren_or_next_member}
```

Comment:
curious where `OPERATOR` is defined.

Reply:
It's defined in the `.flex` file. https://github.com/amzn/ion-intellij-plugin/blob/b3f0951fad29661e9df5e28ade8579ccc60c946f/src/main/kotlin/com/amazon/ion/plugin/intellij/Ion.flex#L79-L82